fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts

```diff
@@ -666,6 +893,57 @@ export class PseudoFWC {
         this.parkBrake.set(SimVar.GetSimVarValue('L:A32NX_PARK_BRAKE_LEVER_POS', 'Bool'));
         this.nwSteeringDisc.set(SimVar.GetSimVarValue('L:A32NX_HYD_NW_STRG_DISC_ECAM_MEMO', 'Bool'));
 
+        /* 22 - AUTOFLIGHT */
+        const fm1DiscreteWord3 = SimVar.GetSimVarValue('L:A32NX_FM1_DISCRETE_WORD_3', 'number');
+        const fm2DiscreteWord3 = SimVar.GetSimVarValue('L:A32NX_FM2_DISCRETE_WORD_3', 'number');
+
+        if (!([2, 3].includes(this.fwcFlightPhase.get()))) {
```

Comment:
Why not use the convenience variable you just made above?

Reply:
Good catch. I did do some search/replace of each incantation, but probably got distracted halfway through by something else I saw 🤣 .